terraform/helm/validator/files/dashboards/components.json

```diff
@@ -2021,7 +2021,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "aptos_core_mempool_index_size{role=~\"$role\",owner=~\"$owner\",index=\"system_ttl\"}",
+          "expr": "core_mempool_index_size{role=~\"$role\",owner=~\"$owner\",index=\"system_ttl\"}",
```

Comment:
Note: I would have changed these counters in a separate PR -- makes the history easier to track 😄 But, no biggie!